signature/fixtures/sigstore_config/test_case_3/2.yaml

```diff
@@ -0,0 +1,5 @@
+default-docker:
+  sigstore: file:///default/sigstore
+docker:
+  example2.com:
+    sigstore: file:///var/lib/containers/sigstore
```

Comment:
Could we add a dummy `sigstore-staging` (https://github.com/containers/image/blob/main/docs/containers-registries.d.5.md) key to ensure the parser will ignore it properly?